_api-reference/cluster-api/cluster-stats.md

```diff
@@ -491,32 +540,32 @@ GET _cluster/stats/nodes/_cluster_manager
-indices.store | Information about the cluster's storage.
-indices.fielddata | Information about the cluster's field data
-indices.query_cache | Data about the cluster's query cache.
-indices.completion | How many bytes in memory are used to complete operations.
-indices.segments | Information about the cluster's segments, which are small Lucene indexes.
-indices.mappings | Mappings within the cluster.
-indices.analysis | Information about analyzers used in the cluster.
-nodes | Statistics about the nodes in the cluster.
-nodes.count | How many nodes were returned from the request.
-nodes.versions | OpenSearch's version number.
-nodes.os | Information about the operating systems used in the nodes.
-nodes.process | The processes the returned nodes use.
-nodes.jvm | Statistics about the Java Virtual Machines in use.
-nodes.fs | The nodes' file storage.
-nodes.plugins | The OpenSearch plugins integrated within the nodes.
-nodes.network_types | The transport and HTTP networks within the nodes.
-nodes.discovery_type | The method the nodes use to find other nodes within the cluster.
-nodes.packaging_types | Information about the nodes' OpenSearch distribution.
-nodes.ingest | Information about the nodes' ingest pipelines/nodes, if there are any.
-total_time_spent | The total amount of download and upload time spent across all shards in the cluster when downloading or uploading from the remote store.
+`nodes` | How many nodes returned in the response.
+`cluster_name` | The cluster's name.
+`cluster_uuid` | The cluster's uuid.
+`timestamp` | The Unix epoch time of when the cluster was last refreshed.
+`status` | The cluster's health status.
+`indices` | Statistics about the indexes in the cluster.
+`indices.count` | How many indexes are in the cluster.
+`indices.shards` | Information about the cluster's shards.
+`indices.docs` | How many documents are still in the cluster and how many documents are deleted.
+`indices.store` | Information about the cluster's storage.
+`indices.fielddata` | Information about the cluster's field data
+`indices.query_cache` | Data about the cluster's query cache.
+`indices.completion` | How many bytes in memory are used to complete operations.
+`indices.segments` | Information about the cluster's segments, which are small Lucene indexes.
+`indices.mappings` | Mappings within the cluster.
+`indices.analysis` | Information about analyzers used in the cluster.
+`nodes` | Statistics about the nodes in the cluster.
+`nodes.count` | How many nodes were returned from the request.
+`nodes.versions` | OpenSearch's version number.
+`nodes.os` | Information about the operating systems used in the nodes.
+`nodes.process` | The processes the returned nodes use.
+`nodes.jvm` | Statistics about the Java Virtual Machines in use.
+`nodes.fs` | The nodes' file storage.
+`nodes.plugins` | The OpenSearch plugins integrated within the nodes.
+`nodes.network_types` | The transport and HTTP networks within the nodes.
```

Comment:
"A list of the"? "Information about the"?

Reply:
```suggestion
`nodes.network_types` | A list of the transport and HTTP networks within the nodes.
```